src/app/new-auth/components/input-field/input-field.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, Input, OnInit, forwardRef } from '@angular/core';
+import { ControlValueAccessor, FormControl, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { AuthValidationService } from 'app/core/auth-validation.service';
+/**
+ * input field for authentication module using
+ * angular material input field and designed
+ * to be used with ngModel
+ *
+ * @param isPwrd: Boolean - make true if the field is for a
```

Comment:
Why are the input fields listed as parameters and commented outside the class?

Reply:
IDK, how should i do them?

Reply:
I wanted to document them somewhere

Reply:
Will something go wrong if it is documented on top of the variable it is referring to?